app/lib/mahonia/datacite_doi_builder.rb

```diff
@@ -0,0 +1,22 @@
+module Mahonia
+  ##
+  # An identifier builder that builds DataCite DOIs in the way recommended by
+  #  the DataCite Metadata Store (MDS) API documentation.
+  #
+  # @see https://support.datacite.org/docs/mds-2
+  class DataciteDoiBuilder < IdentifierBuilder
+    ##
+    # @see IdentifierBuilder#initialize
+    def initialize(prefix: Datacite::Configuration.instance.prefix)
+      super
+    end
+
+    ##
+    # @note hints are ignored
+    #
+    # @see IdentifierBuilder#build
+    def build(*)
+      prefix + '/1234'
```

Comment:
This is confusing to me. Is this a hard coded value?

Reply:
Yes, temporarily. We need to discuss the identifier selection strategy with OHSU.